.yamato/upm-ci.yml

```diff
@@ -236,6 +236,13 @@ test_trigger_hdrp:
     - .yamato/upm-ci.yml#pack
     {% for editor in editors %}
     {% for platform in platforms %}
+    {%- comment -%}
+    # Disable hdrp test in trunk on Mac for now which fails nightly trigger because: 
+    # First the test doesn't test too much. Second, we belive the failure is caused by HDRP in trunk
```

Comment:
Haha, thanks Marie. Sharp eyes. Don't want to correct it now since it will kick off the PR trigger again. Will change it with a future PR.
In the meantime, I am also asking in #dev-hdrp channel about this warning.